From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 5 Sep 2019 14:42:29 +0100
Subject: scsi: qla2xxx: remove redundant assignment to pointer host
Patch-mainline: v5.5-rc1
Git-commit: da6d2965dbdb480f091c42f54a09abe8056282e5
References: bsc#1143706 bsc#1082635 bsc#1123034

The pointer host is being initialized with a value that is never read and
is being re-assigned a little later on. The assignment is redundant and
hence can be removed.

Link: https://lore.kernel.org/r/20190905134229.21194-1-colin.king@canonical.com
Addresses-Coverity: ("Unused value")
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/qla2xxx/qla_target.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/qla2xxx/qla_target.c
+++ b/drivers/scsi/qla2xxx/qla_target.c
@@ -472,7 +472,7 @@ void qlt_response_pkt_all_vps(struct scs
 
 	case IMMED_NOTIFY_TYPE:
 	{
-		struct scsi_qla_host *host = vha;
+		struct scsi_qla_host *host;
 		struct imm_ntfy_from_isp *entry =
 		    (struct imm_ntfy_from_isp *)pkt;
 
